pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -318,14 +386,19 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 	if err != nil {
 		return fmt.Errorf("unable to understand network config: %v", err)
 	}
-	if res.IP4 == nil {
-		return fmt.Errorf("CNI plugin reported no IPv4 address for container %v.", id)
+	//TODO: v1.16 (khenidak) u pdate NET_CONFIG_TEMPLATE to CNI version 0.3.0 or later so
```

Comment:
since we are now targeting 1.16 can this TODO be addressed?

Reply:
Not in this PR, this will be a different PR.